tests/h/schemas/forms/accounts/login_test.py

```diff
@@ -90,6 +90,25 @@ def test_invalid_with_bad_password(
         assert "password" in errors
         assert "Wrong password" in errors["password"]
 
+    @pytest.mark.parametrize(
+        "params,value",
+        [
+            # If a ?username=foobob query param is given then the username field in
+            # the login form is pre-filled with "foobob".
+            ({"username": "foobob"}, "foobob"),
+            # If the foobob query param is missing or has no value then the
+            # username field isn't pre-filled.
+            ({"username": ""}, ""),
```

Comment:
What's the distinction between not filling in the string and filling in empty?

Could it matter for the point of view of focussing?

Reply:
It doesn't matter. Doing an empty string rather than omitting the key entirely, was just easier to code, and made no functional difference